.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+  - repo: https://github.com/pre-commit/mirrors-clang-format
+    rev: v19.1.7
```

Comment:
We should pick a version that we expect people to install with Enzyme, like v15 or so.